website/docs/d/tenant_policies_ipsla_monitoring_policy.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "mso"
+page_title: "MSO: mso_tenant_policies_ipsla_monitoring_policy"
+sidebar_current: "docs-mso-data-source-tenant_policies_ipsla_monitoring_policy"
+description: |-
+Data source for IPSLA Monitoring Policy.
```

Comment:
how to handle abbreviations?

Reply:
I think mentioning it once in the beginning will do. I'll make that change